src/components/FlawAlertsSection.vue

```diff
@@ -48,7 +48,7 @@ const emitExpandFocusedComponent = (parent_uuid: string) => {
 
     <div v-for="alerts,alertType in alertSet" :key="alertType">
       <div v-if="alerts.length">
-        <span v-for="alert in alerts" :key="alert.uuid">
+        <span v-for="(alert, index) in alerts" :key="alert?.uuid || `alert-${index}-${alertType}`">
```

Comment:
Why is this change needed? I mean `alerts` should be always a list, so there will either be some alerts and then each alert should have its own uuid, or the list will be empty, right?